spec/views/crops/search.html.haml_spec.rb

```diff
@@ -11,19 +11,18 @@
       @tomato = FactoryGirl.create(:tomato)
       @roma = FactoryGirl.create(:crop, :name => 'Roma tomato', :parent => @tomato)
       assign(:search, 'tomato')
-      assign(:exact_match, @tomato)
-      assign(:partial_matches, [@roma])
+      assign(:all_matches, [@tomato, @roma])
       render
     end
 
     it "shows exact matches" do
-      assert_select "div#exact_match" do
+      assert_select "div#all_matches" do
         assert_select "a[href=#{crop_path(@tomato)}]"
       end
     end
 
-    it "shows partial matches" do
-      assert_select "div#partial_matches" do
+    it "shows all matches" do
```

Comment:
the issue here is still that it shows partial matches. i think the text as it previously stood is clearer. after all, we're not testing that it shows *all* the matches, are we? just that it shows at least one partial match.